### ISSUE TITLE
Ability to exclude zero-values from tooltip

### ISSUE THREAD

**Issue by xobotyi** [apexcharts/apexcharts.js#3700]
## Summary

A have a dynamic data with unknown amount of series for a given time range.
As apexcharts not allowing to draw null vlaues as 0 - i had to transform missing data to be 0, but now 0 values appear in tooltip.

## API Changes

Add an `undefined` as tooltip value formatter that means - do not display this item in tooltip.

Or maybe add ability to draw null values as some others values (0 for me). 

## Intended Use Case

As for now i was not able to find how to exclude zero-values from tooltip of area charts (or, actually, draw missing values as 0). This leads to very high tooltip that occasionally causes page to have a scrollbar and this triggers tooltip to hide.

![image](https://user-images.githubusercontent.com/6178739/224795204-d8fa6193-fd1c-44c7-a2c7-7d3b1b234f6d.png)

After trying to make nulls work figured that nulls are also displayed within tooltip🤔
![image](https://user-images.githubusercontent.com/6178739/224798753-4a46890f-aa1f-4e5c-98bf-e34bb0e6aad3.png)

**Comment by pvh-archifact** [comment]
This can be easy  resolved using the tooltip formatter:

`tooltip: {
  y: {
    formatter: function(value, { series, seriesIndex, dataPointIndex, w }) {
    if(Value){
      return value
}
    }
  }
}`

**Comment by xobotyi** [comment]
The type of a formatter function does not expect undefined to be returned.
There is an issue in types then.

**Comment by xobotyi** [comment]
Your codepen is JS-only, im talking about issue with types.

https://github.com/apexcharts/apexcharts.js/blob/main/types/apexcharts.d.ts#L940

It is clearly typed that it always expects a value to be returned.

**Comment by pvh-archifact** [comment]
Change 'X' to Y

**Comment by pvh-archifact** [comment]
I do not know why you see undefined, if i test it, it either displays 0 or null not 'undefined'

**Comment by xobotyi** [comment]
because i do return it (just like your yesterday example)
once again - the goal is to exclude zero\null values from tooltip

**Comment by melbertdp** [comment]
Hey, we also experienced this issue when we upgraded to latest version.
this is working on version
"apexcharts": "3.24.0",
"react-apexcharts": "1.3.7",
![image](https://user-images.githubusercontent.com/110758934/227157030-25b316e5-d53e-4c7d-9483-9d67cfc63777.png)

but not on version
"apexcharts": "3.37.2",
"react-apexcharts": "1.4.0",
![image](https://user-images.githubusercontent.com/110758934/227157281-23816929-a8f0-45f9-a50c-ba1f5c91ae50.png)

**Comment by pvh-archifact** [comment]
Is this what you are looking for?

[https://codepen.io/PlippiePlop/pen/RwYqJjM](url)

**Comment by xobotyi** [comment]
yes, but im definely don't want to create my own custom tooltip for that purpose =)
